src/applications/simple-forms/21-10210/sass/10210-lay-witness-statement.scss

```diff
@@ -29,10 +29,28 @@
   }
 }
 
-.schemaform-label[for*="root_witnessRelationship"] > .form-required-span,
-#root_witnessOtherRelationshipToVeteran-label > .schemaform-required-span {
+#root_witnessRelationshipToClaimant-label {
```

Comment:
FYI, we've communicated with DST about these issues and open defects about us needing to use custom CSS settings for these to match our existing forms. they're correcting them incrementally, especially for the v3 version of web components (ui:options: { uswds: true }}  or <va-webcomponent-name usdws /> (probably not ready to use yet)

Reply:
Thx, @rhasselle-oddball !  Yeah, I think we should consider using `path="shadowDomElement"` attributes in our components to support CSS customizations via `::part(shadowDomElement)` pseudo-selector.  At one point, I had to hide a label with the ugly `clip-path` CSS-rule -- which prob'ly wouldn't have hidden it from screenreaders!